tests/test_gates.py

```diff
@@ -37,58 +37,56 @@
 @pytest.fixture
 def op(op_name):
     ops_list = {
-        "RX": qml.RX(0.123, wires=0),
-        "RY": qml.RY(1.434, wires=0),
-        "RZ": qml.RZ(2.774, wires=0),
-        "S": qml.S(wires=0),
-        "SX": qml.SX(wires=0),
-        "T": qml.T(wires=0),
-        "CNOT": qml.CNOT(wires=[0, 1]),
-        "CZ": qml.CZ(wires=[0, 1]),
-        "CY": qml.CY(wires=[0, 1]),
-        "SWAP": qml.SWAP(wires=[0, 1]),
-        "ISWAP": qml.ISWAP(wires=[0, 1]),
-        "SISWAP": qml.SISWAP(wires=[0, 1]),
-        "SQISW": qml.SQISW(wires=[0, 1]),
-        "CSWAP": qml.CSWAP(wires=[0, 1, 2]),
-        "PauliRot": qml.PauliRot(0.123, "Y", wires=0),
-        "IsingXX": qml.IsingXX(0.123, wires=[0, 1]),
-        "IsingYY": qml.IsingYY(0.123, wires=[0, 1]),
-        "IsingZZ": qml.IsingZZ(0.123, wires=[0, 1]),
-        "Identity": qml.Identity(wires=0),
-        "Rot": qml.Rot(0.123, 0.456, 0.789, wires=0),
-        "Toffoli": qml.Toffoli(wires=[0, 1, 2]),
-        "PhaseShift": qml.PhaseShift(2.133, wires=0),
-        "ControlledPhaseShift": qml.ControlledPhaseShift(1.777, wires=[0, 2]),
-        "CPhase": qml.CPhase(1.777, wires=[0, 2]),
-        "MultiRZ": qml.MultiRZ(0.123, wires=[0, 3]),
-        "MultiRZ": qml.MultiRZ(0.456, wires=[1, 2, 3]),
-        "MultiRZ": qml.MultiRZ(0.789, wires=[1, 2]),
-        "CRX": qml.CRX(0.836, wires=[2, 3]),
-        "CRY": qml.CRY(0.721, wires=[2, 3]),
-        "CRZ": qml.CRZ(0.554, wires=[2, 3]),
-        "Hadamard": qml.Hadamard(wires=0),
-        "PauliX": qml.PauliX(wires=0),
-        "PauliY": qml.PauliY(wires=0),
-        "PauliZ": qml.PauliZ(wires=0),
-        "CRot": qml.CRot(0.123, 0.456, 0.789, wires=[0, 1]),
-        "DiagonalQubitUnitary": qml.DiagonalQubitUnitary(np.array([1.0, 1.0j]), wires=1),
-        "ControlledQubitUnitary": qml.ControlledQubitUnitary(
-            np.eye(2) * 1j, wires=[0], control_wires=[2]
-        ),
-        "MultiControlledX": qml.MultiControlledX(
-            control_wires=[0, 1], wires=2, control_values="01"
-        ),
-        "SingleExcitation": qml.SingleExcitation(0.123, wires=[0, 3]),
-        "SingleExcitationPlus": qml.SingleExcitationPlus(0.123, wires=[0, 3]),
-        "SingleExcitationMinus": qml.SingleExcitationMinus(0.123, wires=[0, 3]),
-        "DoubleExcitation": qml.DoubleExcitation(0.123, wires=[0, 1, 2, 3]),
-        "DoubleExcitationPlus": qml.DoubleExcitationPlus(0.123, wires=[0, 1, 2, 3]),
-        "DoubleExcitationMinus": qml.DoubleExcitationMinus(0.123, wires=[0, 1, 2, 3]),
-        "QFT": qml.QFT(wires=0),
-        "QubitSum": qml.QubitSum(wires=[0, 1, 2]),
-        "QubitCarry": qml.QubitCarry(wires=[0, 1, 2, 3]),
-        "QubitUnitary": qml.QubitUnitary(np.eye(2) * 1j, wires=0),
+        "RX": [qml.RX, [], {"phi": 0.123, "wires": [0]}],
```

Comment:
Nice that you brought this update from lightning too! :slightly_smiling_face:

Reply:
Thanks for the nice reference @AmintorDusko !